src/main/java/blackjack/domain/Person.java

```diff
@@ -0,0 +1,44 @@
+package blackjack.domain;
+
+import blackjack.dto.CardInfo;
+import blackjack.dto.NameInfo;
+import blackjack.dto.ScoreInfo;
+
+public abstract class Person {
+
+    public static final int INIT_CARD_COUNT = 2;
+
+    final String userName;
+    final Cards cards;
+
+    public Person(final String userName,final Cards cards) {
```

Comment:
`cmd + opt + L`을 이용해서 포맷팅을 하면 좋을 것 같아요~

```suggestion
    public Person(final String userName, final Cards cards) {
```